org.eclipse.jdt.core.compiler.batch/scripts/.gitignore

```diff
@@ -0,0 +1 @@
+*.class
```

Comment:
Not sure if this is supposed to be merged in -

Reply:
this is intentional; I would like to avoid staging `GenerateParserScript.class`